Source/Scene/ClippingPlanesCollection.js

```diff
@@ -0,0 +1,189 @@
+define([
+    '../Core/BoundingSphere',
+    '../Core/buildModuleUrl',
+    '../Core/Cartesian3',
+    '../Core/Cartographic',
+    '../Core/Color',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/Intersect',
+    '../Core/Matrix4',
+    '../Core/Plane'
+], function(
+    BoundingSphere,
+    buildModuleUrl,
+    Cartesian3,
+    Cartographic,
+    Color,
+    defaultValue,
+    defined,
+    Intersect,
+    Matrix4,
+    Plane) {
+    'use strict';
+
+    /**
+     * Contains the options to specify a set of clipping planes. Clipping planes selectively disable rendering on an object on the outside of the specified list of {@link Plane}.
```

Comment:
Keep the reference doc direct.  Remove "Contains the options to"

It just "Specifies a set..."